src/ngraph/runtime/cpu/cpu_emitter.cpp

```diff
@@ -3103,6 +3103,90 @@ namespace ngraph
                 }
             }
 
+            template <>
+            void CPU_Emitter::EMITTER_DECL(ngraph::op::ConvolutionBiasAdd)
+            {
+                auto convolution = static_cast<const ngraph::op::ConvolutionBiasAdd*>(node);
+
+                auto arg0_shape = args[0].get_shape();
+                auto arg1_shape = args[1].get_shape();
+                auto arg2_shape = args[2].get_shape();
+                auto arg3_shape = args[3].get_shape();
```

Comment:
I think this part could use a comment saying that MKLDNN expects `add_input` data in `out[0]` so args3_shape isn't used. Also, why aren't you getting a compilation error here since `arg3_shape` doesn't seem to be used anywhere.